server-side has() can only check for custom perms (#1836)
clerk/clerk-docs: docs/organizations/roles-permissions.mdx

Permissions grant users privileged access to resources and operations, like creating and deleting. Clerk supports two types of permissions: System and Custom.

### System permissions

> Clerk has a set of system permissions that power [Clerk's Frontend API](/docs/reference/frontend-api){{ target: '_blank' }} and [organization-related Clerk components](/docs/components/overview#organization-components). They are a baseline set of permissions that Clerk needs to operate functionally.

Reviewer: ```suggestion
Clerk provides a set of system permissions that power [Clerk's Frontend API](/docs/reference/frontend-api){{ target: '_blank' }} and [organization-related Clerk components](/docs/components/overview#organization-components). These permissions serve as the baseline required for Clerk to operate effectively.
```

- Read domains (`org:sys_domains:read`)
- Manage domains (`org:sys_domains:manage`)

You can assign these system permissions to any role.

> [!WARNING]

> > System permissions are not included in [session claims](/docs/backend-requests/resources/session-tokens#default-session-claims). To do permission-checks on the server-side, you must [create custom permissions](/docs/organizations/create-roles-permissions).

Reviewer: ```suggestion
> System permissions aren't included in [session claims](/docs/backend-requests/resources/session-tokens#default-session-claims). To check permissions on the server-side, you must [create custom permissions](/docs/organizations/create-roles-permissions).
```
